src/client.ts

```diff
@@ -36,11 +36,17 @@ export class PublicClient {
 }
 
 export class Client extends PublicClient {
-  public user: User;
+  public user?: User;
+
+  isAPIKey(authorization: string): boolean {
+    return authorization.startsWith('VF.');
+  }
 
   constructor({ clientKey, apiEndpoint, authorization, options }: Omit<ClientOptions, 'authorization'> & { authorization: string }) {
     super({ clientKey, apiEndpoint, authorization, options });
 
-    this.user = new User(authorization);
+    if (authorization && !this.isAPIKey(authorization)) {
+      this.user = new User(authorization);
```

Comment:
have you searched where we use `user` in the other services? (general, google, alexa)-service I suppose uses it somewhere. We might want to make sure this being `undefined` doesn't crash. Maybe default to empty obj?